src/main/java/com/kodilla/ecommercee/repository/OrderRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Order;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+
+/**
+ * Author Kamil Seweryn
+ */
+@Transactional
```

Comment:
Jarema wspominał o tym że @Transactional lepiej dawać w serwisach

Reply:
Na razie nie mam serwisu, więc zostawię tutaj, a jak ktoś zrobi serwis to usuniemy. :-)

Reply:
Dobra. Ja zrobię serwis